helm/bankapp/values.yaml

```
@@ -0,0 +1,54 @@
+    targetPort: 8080
+    nodePort: 30080 
+
+
+# headless svc configuration for db
+mysql_svc:
+    port: 3306
+    targetPort: 3306
+
+# HPA minimun and maximum pods and average cpu utilization
+hpa:
+    min_replica: 1
```

yamllint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)